app/controllers/application_controller/buttons.rb

```diff
@@ -624,7 +624,7 @@ def ab_button_reset
     drop_breadcrumb(:name => _("Edit of Button"), :url => "/miq_ae_customization/button_edit")
     @lastaction = "automate_button"
     @layout = "miq_ae_automate_button"
-    replace_right_cell(:nodetype => "button_edit")
+    replace_right_cell(:action => "button_edit")
```

Comment:
@lgalis changing this to pass in :action instead of :nodetype might break custom button editor in Customization explorer, i think we should pass in both keys here in order for this to work in both explorers. Will need to test this though. same suggestion on line 496 as well

Reply:
@lgalis verified in UI, both explorers seems to be working fine, ignore my previous comment.

Reply:
@h-kataria - thank you - I have tried the button editor in both the service as well as auomation before creating the PR.